colibre/auto_plotter/black_holes.yml

```
@@ -123,7 +123,7 @@ stellar_veldisp_black_hole_mass_10:
       units: km/s
   metadata:
     title: Stellar Velocity Dispersion-Black Hole Mass relation (10 kpc)
```

I think it would be good to indicate that the plotted velocity dispersion is the **LOS** velocity dispersion.

If you agree with this suggestion, then please also update the titles of the other figures where the LOS velocity dispersion is shown.

```suggestion
    title: LOS Stellar Velocity Dispersion-Black Hole Mass relation (10 kpc)
```